docs/getting-started/cronos-testnet.md

```diff
@@ -31,6 +31,19 @@ The latest Crypto.org Chain Testnet has been named as **Cronos**.
 
 This is a detailed documentation for setting up a Validator or a full node on Crypto.org Cronos testnet `cronostestnet_338-3`.
 
+
+Before we start, please note that there was new binary upgrade:
+
+- Start the node with the older binary version v0.6.0;
+- Sync-up with the blockchain until it reaches the target upgrade block height `1553700`;
```

Comment:
```
panic: UPGRADE "v0.7.0" NEEDED at height: 1553700
```

will be the expected error messages when we hit that block, we can also add it as a reference.